package.json

```diff
@@ -39,48 +39,48 @@
     "react-fast-compare": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
     "@adobe/aem-spa-component-mapping": "~1.1.0",
-    "@adobe/aem-spa-page-model-manager": "~1.3.6"
+    "@adobe/aem-spa-page-model-manager": "~1.3.10",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
   },
   "devDependencies": {
-    "@adobe/aem-spa-component-mapping": "~1.1.0",
-    "@adobe/aem-spa-page-model-manager": "~1.3.6",
-    "@adobe/eslint-config-editorxp": "^1.0.5",
+    "@adobe/aem-spa-component-mapping": "file:../aem-spa-component-mapping",
```

Comment:
will be updated, once https://github.com/adobe/aem-spa-component-mapping/pull/39/files is merged